linux/user-and-file-management/linux-intro/hello-linux.md

```diff
@@ -0,0 +1,51 @@
+---
+author: kapnobatai136
+
+aspects:
+  - introduction
+
+type: normal
+
+category: must-know
+
+tags:
+  - introduction
+  - linux
+
+links:
+  - '[Install Ubuntu](https://ubuntu.com/tutorials/tutorial-install-ubuntu-desktop#1-overview){website}'
+  - '[Try Ubuntu](https://ubuntu.com/tutorials/try-ubuntu-before-you-install#1-getting-started){website}'
+
+---
+
+# Hello Linux
+
+---
+## Content
+
+All this talk, and we still haven't shown you how to use Linux 😅.
+
+> Note: we're going to assume you have an installed Linux distribution. If you don't, check out the *Learn More* section for Ubuntu installation guides.
+
+![printing-hello-linux-in-shell](https://img.enkipro.com/b293f5a296e067d80f9dc7e77c7072d3.gif)
+
+This is the Linux terminal[1]! It's where you'll be typing most of the commands.
+
+In computer science, what you see in the example is called *printing*, a fancy word for displaying text. 
+
+> **Note:** We'll explain how printing works in the next workout.
```

Comment:
Why don't we explain the "shell"-"terminal" relationship part here? The next insight feels quite crowded without any other visual element besides text?

Reply:
We should only do this if it's easy to keep the same quality of reading flow, but no need to push it.

Reply:
move the comparison here. the flow is better this way

Reply:
Cool 👍 

Remember to mention that a shell isn't exactly the same as the terminal and that it's a program running in the terminal.